api/Trace/Events.php

```diff
@@ -10,10 +10,10 @@ interface Events extends \IteratorAggregate, \Countable
      * Adding an event should not invalidate nor change any existing iterators.
      * @param string $name
      * @param Attributes|null $attributes
-     * @param string|null $timestamp
+     * @param int|null $timestamp
      * @return Events Must return $this to allow setting multiple attributes at once in a chain.
      */
-    public function addEvent(string $name, ?Attributes $attributes = null, ?string $timestamp = null): Events;
+    public function addEvent(string $name, ?Attributes $attributes = null, ?int $timestamp = null): Events;
```

Comment:
I'm wondering if we should make this `addEvent` timestamp required as well.

Reply:
Not as important to me, but I'd at least put its parameters in the same order in case we decide to make it required.